src/main/java/io/fabric8/vertx/maven/plugin/utils/FileUtils.java

```diff
@@ -15,6 +18,21 @@
  */
 public class FileUtils {
 
+
+    public static Set<Path> includedDirs(MavenProject project, Optional<List<String>> includes) {
+
+        //TODO Handle resource resource roots and compile roots from respective plugins and includes
```

Comment:
If I understand the comment right, this is still to be implemented ? (i.e. the usage of `includes` which btw is always a null optional if not set as plugin parameter (has no default))